lib/helpers.js

```diff
@@ -19,4 +22,17 @@ module.exports = {
             .request(method, url, request, options);
     },
 
+    readLaunchesFromFile() {
```

Comment:
will be nice to have test for both readLaunchesFromFile\saveLaunchIdToFile

Reply:
I am going to add tests for these functions a bit later. Currently I don't have enough time due to sprint issues.